strongbox-commons/src/main/java/org/carlspring/strongbox/booters/PropertiesBooter.java

```diff
@@ -28,9 +28,6 @@
     @Value("${strongbox.etc}")
     private String etcDirectory;
 
-    @Value("${strongbox.temp}")
-    private String tempDirectory;
```

Comment:
@strongbox/core-developers : Are we sure we really want to drop this? This should typically be under the `strongbox-vault`. It might not be used currently, but is there a benefit in dropping this?